draft-wood-ohai-unreliable-ohttp.md

```diff
@@ -89,17 +89,18 @@ privacy. We describe these motivating properties below.
 
 ## Gateway Performance and Security
 
-{{STAR}} is a proposed system for privacy-preserving data collection aimed at the
-heavy hitters problem. For privacy reasons, it is important that client reports,
-containing their individual measurements, are separated from any client identifying
-information, including their IP address. STAR can use OHTTP to send client reports,
-but it requires the Oblivious Gateway Resource to produce an encrypted acknowledgement
-to the clients for every report.
-
-Depending on the Oblivious Gateway Resource implementation and scale of deployment,
-this can lead to reduced performance. It also requires the Oblivious Gateway Resource to
-have access to the private key necessary to process the Encapsulated Request carrying
-a report and produce a response.
+In privacy-preserving data collection systems, it is important that client reports
```

Comment:
Do you mean that the PRIO shuffler or aggregator breaks the link between the client IP address and their report?

More generally I was trying to get away from 'This is only motivated by this one proposal from another working group.' It's a good example but I didn't want to lead with Star. What about:
> In privacy-preserving data collection systems, it can be important that...

Reply:
> Do you mean that the PRIO shuffler or aggregator breaks the link between the client IP address and their report?

Nope! In Prio, each report is basically encrypted and never revealed in plaintext to an aggregator, so they leaking the IP address is not a problem.

Your proposed alternate sounds good. I might say "In _some_ privacy-preserving data collection systems, it can be important that..."